deps.edn

```diff
@@ -10,7 +10,10 @@
            ring/ring-core                        {:mvn/version "1.11.0"}
            hiccup/hiccup                         {:mvn/version "2.0.0-RC2"}
            bidi/bidi                             {:mvn/version "2.1.6"}
-           it.burning/cron-expression-descriptor {:mvn/version "1.2.10"}}
+           it.burning/cron-expression-descriptor {:mvn/version "1.2.10"}
+           com.cognitect.aws/api       {:mvn/version "0.8.710-beta01"}
```

Comment:
Did you explore the trade-offs between `cognitect-labs/aws-api` and `mcohen01/amazonica`? Why did you choose the former over the latter?